x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/types.ts

```diff
@@ -101,3 +105,75 @@ export type CrawlerDomainValidationStepName =
   | 'networkConnectivity'
   | 'indexingRestrictions'
   | 'contentVerification';
+// See SharedTogo::Crawler::Status for details on how these are generated
```

Comment:
Not sure if it's confusing to reference something in a separate repo that open-source contributors can't access 🤔  should we consider removing this comment? (No strong feelings either way)

Reply:
I think this is a very useful comment for developers working on this (who, lets be real, is exclusively the App Search FE and will likely continue to be mostly us for the foreseeable future)